dqgen/services/__init__.py

```diff
@@ -16,20 +16,35 @@
 QUERIES_TEMPLATES = Environment(loader=PackageLoader("dqgen.resources", "query_templates"))
 PATH_TO_STATIC_FOLDER = pathlib.Path(__file__).parent.parent / "resources" / "html_templates" / "static"
 
-CLASSES_OPERATION_TEMPLATE_MAPPING = {"added_instance": QUERIES_TEMPLATES.get_template("instance_additions.rq"),
-                                      "deleted_instance": QUERIES_TEMPLATES.get_template("instance_deletions.rq")}
-PROPERTIES_OPERATION_TEMPLATE_MAPPING = {"added_property": QUERIES_TEMPLATES.get_template("property_additions.rq"),
-                                         "deleted_property": QUERIES_TEMPLATES.get_template("property_deletions.rq"),
-                                         "updated_property": QUERIES_TEMPLATES.get_template("property_value_updates.rq"),
-                                         "moved_property": QUERIES_TEMPLATES.get_template("movement_cross_instance.rq"),
-                                         "changed_property": QUERIES_TEMPLATES.get_template("movement_cross_property.rq")
-                                         }
+CLASSES_OPERATION_TEMPLATE_MAPPING = {
```

Comment:
I like this registry pattern. 

This variant (implementation), has a drawback: you need to remember the keys (no autocomplete is available for string keys). 

To overcome this limitation you can do either:
- define class  properties  (i.e. a method annotated with @property decorator) OR
- define String Constants that you use as keys in your current dict registry.